queue/src/main/resources/solr-doc-store-queue/migration/V1_2__Classifier.sql.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE queue
+  ADD COLUMN classifier TEXT NOT NULL;
+
+ALTER TABLE queue_error
+  ADD COLUMN classifier TEXT NOT NULL;
```

Comment:
Der mangler en default, ligesom `ALTER TABLE bibliographicSolrKeys ALTER COLUMN classifier SET NOT NULL;`. Tror det er derfor at når jeg kører det lokalt, at den crasher.